script.js

```diff
@@ -521,3 +521,14 @@ closebtn.addEventListener('click', () => {
   document.querySelector('.form_button').style.display = 'block';
   document.querySelector('.frame-skill').style.display = 'flex';
 });
+
+const userEmailBox = document.getElementById('EmailValidation');
+
+document.querySelector('#FormValidation').addEventListener('submit', (event) => {
+  // event.preventDefault();
```

Comment:
- [OPTIONAL] Following the best practices, it is not advisable to leave un-used code as a comment on your codebase, it makes your code base look unprofessional. I'll suggest you remove this. :smile: :+1:

Reply:
thanks for remarks, I will keep you advice in mind for the future Projects.